valhalla/mjolnir/valhalla_add_elevation_utils.h

```diff
@@ -1,46 +0,0 @@
-#pragma once
-
-#include <deque>
-#include <string>
-#include <unordered_set>
-
-#include <boost/property_tree/ptree.hpp>
-
-#include "baldr/graphid.h"
-#include "baldr/graphreader.h"
-#include "baldr/graphtile.h"
-#include "filesystem.h"
-#include "midgard/logging.h"
-
-namespace valhalla {
-namespace mjolnir {
-
-std::deque<baldr::GraphId> get_tile_ids(const boost::property_tree::ptree& pt,
```

Comment:
this header was a bit unnecessary, was only used in valhalla_add_elevation (as the header name suggests..)